community/openvpn/build

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+autoreconf -i
```

Comment:
Is there no way to avoid `autoreconf`? ie a release tarball of some kind?

Reply:
When make doesn't work without ./configure, and when there is only a configure.ac file I have to run autoreconf to create configure so we can make the magic happen. If there is a better way let me know??

Reply:
Generally you'll want to find a better source than directly fetching it from GitHub tags (but still from its official site). Try to check the official OpenVPN site, or find the source from Void's `template` for this package.

Reply:
thanks